tools/cli/commands.js

```diff
@@ -3362,7 +3362,7 @@ const setupBenchmarkSuite = async (profilingPath) => {
   process.env.GIT_TERMINAL_PROMPT = 0;
 
   const repoUrl = "https://github.com/meteor/performance";
-  const branch = "v3.2.0";
+  const branch = "v3.3.0";
```

Comment:
is not there a way to always get the latest branch?

Reply:
Yes, but I decided to version it, since it can be risky and even small changes can break the command partially. It actually happened a few times.